src/Interfaces/AbstractAffineMap.jl

```diff
@@ -0,0 +1,303 @@
+
+### Input
+
+- `am`       -- affine map
+- `cond_tol` -- (optional) tolerance of matrix condition (used to check whether
+                the matrix is invertible)
+
+### Output
+
+`true` iff the affine map is bounded.
+
+### Algorithm
+
+We first check if the matrix is zero or the wrapped set is bounded.
+If not, we perform a sufficient check whether the matrix is invertible.
+If the matrix is invertible, then the map being bounded is equivalent to the
+wrapped set being bounded, and hence the map is unbounded.
+Otherwise, we check boundedness via [`isbounded_unit_dimensions`](@ref).
+"""
+function isbounded(am::AbstractAffineMap; cond_tol::Number=DEFAULT_COND_TOL)
+    M = matrix(am)
+    if iszero(M) || isbounded(set(am))
+        return true
+    end
+    if isinvertible(M; cond_tol=cond_tol)
+        return false
+    end
+    return isbounded_unit_dimensions(am)
+end
+
+"""
+    ∈(x::AbstractVector{N}, am::AbstractAffineMap{N}) where {N<:Real}
+
+Check whether a given point is contained in the affine map of a convex set.
+
+### Input
+
+- `x`  -- point/vector
+- `am` -- affine map of a convex set
+
+### Output
+
+`true` iff ``x ∈ am``.
+
+### Algorithm
+
+Note that ``x ∈ M⋅S ⊕ v`` iff ``M^{-1}⋅(x - v) ∈ S``.
+This implementation does not explicitly invert the matrix, which is why it also
+works for non-square matrices.
+
+### Examples
+
+```jldoctest
+julia> am = AffineMap([2.0 0.0; 0.0 1.0], BallInf([1., 1.], 1.), [-1.0, -1.0]);
+
+julia> [5.0, 1.0] ∈ am
+false
+
+julia> [3.0, 1.0] ∈ am
+true
+```
+
+An example with a non-square matrix:
+
+```jldoctest
+julia> B = BallInf(zeros(4), 1.);
+
+julia> M = [1. 0 0 0; 0 1 0 0]/2;
+
+julia> [0.5, 0.5] ∈ M*B
+true
+```
+"""
+function ∈(x::AbstractVector{N}, am::AbstractAffineMap{N}) where {N<:Real}
+    return matrix(am) \ (x - vector(am)) ∈ set(am)
+end
+
+"""
+    vertices_list(am::AbstractAffineMap{N};
+                  [apply_convex_hull]::Bool) where {N<:Real}
+
+Return the list of vertices of a (polyhedral) affine map.
+
+### Input
+
+- `am`                -- affine map
+- `apply_convex_hull` -- (optional, default: `true`) if `true`, apply the convex
+                         hull operation to the list of vertices transformed by
+                         the affine map
+
+### Output
+
+A list of vertices.
+
+### Algorithm
+
+This implementation computes all vertices of `X`, then transforms them through
+the affine map, i.e. `x ↦ M*x + v` for each vertex `x` of `X`. By default, the
+convex hull operation is taken before returning this list. For dimensions three
+or higher, this operation relies on the functionality through the concrete
+polyhedra library `Polyhedra.jl`.
+
+If you are not interested in taking the convex hull of the resulting vertices
+under the affine map, pass `apply_convex_hull=false` as a keyword argument.
+
+Note that we assume that the underlying set `X` is polyhedral, either concretely
+or lazily, i.e. there the function `vertices_list` should be applicable.
```

Comment:
can we check that `vertices_list` is applicable and send an error message otherwise? or define and use `ispolytopic(X)` from [this comment](https://github.com/JuliaReach/LazySets.jl/issues/996#issuecomment-586589969)